python/pyspark/rdd.py

```diff
@@ -353,8 +365,8 @@ def sample(self, withReplacement, fraction, seed=None):
         :param seed: seed for the random number generator
 
         >>> rdd = sc.parallelize(range(100), 4)
-        >>> rdd.sample(False, 0.1, 81).count()
-        10
+        >>> 9 <= rdd.sample(False, 0.1, 81).count() <= 11
```

Comment:
Is this change due to different RNG behavior between Python 2 and 3 (e.g. were we testing implementation-specific details here)?